core/dbt/config/project.py

```diff
@@ -310,11 +310,20 @@ def check_config_path(self, project_dict, deprecated_path, exp_path):
                 raise DbtProjectError(
                     msg.format(deprecated_path=deprecated_path, exp_path=exp_path)
                 )
-            deprecations.warn(
-                f"project-config-{deprecated_path}",
-                deprecated_path=deprecated_path,
-                exp_path=exp_path,
-            )
+            # this field has been renamed
+            if exp_path:
+                deprecations.warn(
+                    f"project-config-{deprecated_path}",
+                    deprecated_path=deprecated_path,
+                    exp_path=exp_path,
+                )
+            # this field is no longer supported, but many projects may specify it with the default value
+            # if so, let's only raise this deprecation warning if they set a custom value
```

Comment:
❗I welcome disagreement ❗

Reply:
I'm fine with this. Why rouse them to action when there's really no need. It's just a syntactic artifact. At most, you could tell people "hey this doesn't even do anything anymore tbh" but also idk if it really impacts anyone.